packages/core/src/state-management/state-manager.ts

```diff
@@ -30,6 +30,8 @@ export class StateManager {
     public anchorService: AnchorService,
     public conflictResolution: ConflictResolution,
     private readonly logger: DiagnosticsLogger,
+    private readonly get: (docId: DocID) => Promise<RunningState | undefined>,
```

Comment:
Would be nice to have a comment clarifying what `get` and `load` are for and how they're different

Reply:
Maybe just renaming it to `getFromRepository` would make it clearer?

Reply:
`fromMemoryOrStore` ?

Reply:
sure, that works

Reply:
Done